config/config.go

```diff
@@ -86,9 +86,11 @@ const (
 	EnvLdapGroupFilter        = "LDAP_GROUP_FILTER"
 	EnvLdapBase               = "LDAP_BASE"
 	EnvLdapBindDN             = "LDAP_BIND_DN"
-	EnvLdapTokenExpirationMin = "LDAP_TOKEN_EXPIRATION_MIN"
 	EnvLdapRoleFilter         = "LDAPROLEFILTER"
 	EnvLdapSearchFilter       = "LDAP_SEARCH_FILTER"
+	EnvLdapMailIdKey          = "LDAP_MAIL_ID_KEY"
+	EnvLdapUserIdKey          = "LDAP_USER_ID_KEY"
+	EnvLdapMemeberOfKey       = "LDAP_MEMBER_OF_KEY"
```

Comment:
"Memeber" is a misspelling of the word "Member"
```suggestion
	EnvLdapMemberOfKey       = "LDAP_MEMBER_OF_KEY"
```
So you will need to rename this and fix all references to it.